.travis.yml

```diff
@@ -3,8 +3,20 @@ go:
     - 1.7
     - 1.8
     - tip
-script: go test -v -randtests 1000 ./...
+before_script:
+    - PKGS=$(go list ./... | grep -v /vendor/)
+    - GO_FILES=$(find . -iname '*.go' -not \( -path '*/vendor/*' -prune \))
+    - go get -u honnef.co/go/tools/cmd/gosimple
+    - go get -u honnef.co/go/tools/cmd/staticcheck
+    - go get -u honnef.co/go/tools/cmd/unused
+script:
+    - test -z $(gofmt -s -l $GO_FILES)
+    - go vet $PKGS
+    - staticcheck $PKGS
+    - gosimple $PKGS
```

Comment:
I don't even know why this bugs me 😅, but could you keep the order of `gosimple`, `staticcheck` and `unused` commands the same as they are in the go get call(s). Switching `staticcheck` with `gosimple` should be perfect.